README.md

```diff
@@ -31,37 +27,10 @@ cd dkls
 wasm-pack build --release --target web
 ```
 
-#### Step 2
+### Step 2
 
-From the `pkg` subfolder in `dkls`:
+Copy the files from `dkls/pkg` (except the WASM and the package.json) to `packages/tss-lib`.
 
-Copy the following files:
+### Step 3
 
-```
-dkls.d.ts
-dkls_bg.wasm.d.ts
-```
-
-into this repositories' path `packages/tss-lib`.
-
-#### Step 3
-
-Copy contents of `dkls.js` into `packages/tss-lib/browser.js`.
-
-#### Step 4
-
-Revert any changes to the following functions and exports in `browser.js`:
-
-```
-function initSync(module)
-async function init(input)
-export { initSync, finalizeInit, getImports }
-```
-
-#### Step 5
-
-Copy and rename `dkls_bg.wasm` to `client.wasm` and overwrite `packages/tss-lib/wasm/client.wasm` with it.
-
-#### Step 6
-
-Publish changes to nmp for packages tss-lib and tss-client.
+Copy the WASM file from `dkls/pkg` to `packages/tss-lib-wasm`.
```

Comment:
Why do this separately?

Reply:
The goal is to package the WASM separately due to lib size constraints of mpc core kit, see above.